test/e2e/specs/chaptersPage.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { vpTest } from '../fixtures/vpTest';
+import { expect, test } from '@playwright/test';
+import { waitForPageToLoadWithTimeout } from '../src/helpers/waitForPageToLoadWithTimeout';
+import { getLinkByName } from '../testData/pageLinksData';
+import { ExampleLinkName } from '../testData/ExampleLinkNames';
+
+// Link to chapters page
```

Comment:
I think its already clear from the code, i will refrain adding comments when the code makes it clear ... this should also be the target, a self descriptive code 
```suggestion
```